charts/kafka-connect-redis-sink/templates/deployment.yaml

```diff
@@ -214,23 +214,30 @@ spec:
         - name: CONNECTOR_TASKS_MAX
           value: {{ .Values.replicaCount | quote }}
         - name: CONNECTOR_TOPICS
-          value: {{ .Values.topics | quote }}    
+          value: {{ .Values.topics | quote }}
         - name: CONNECTOR_CONNECT_REDIS_HOST
-          value: {{ .Values.host | quote }}          
+          value: {{ .Values.host | quote }}
         - name: CONNECTOR_CONNECT_REDIS_PORT
           value: {{ .Values.port | quote }}
+        {{- if .Values.password }}
+        # only create SECRET_CONNECT_REDIS_PASSWORD if auth token supplied
         - name: SECRET_CONNECT_REDIS_PASSWORD
           valueFrom:
             secretKeyRef:
               name: {{ include "fullname" . }}
               key:  "connect.redis.password"
+        {{- else }}
+        # Hack so that lenses-cli will create /etc/kafka-connect/secrets.properties
+        - name: SECRET_I_DONT_MATTER
```

Comment:
not the best solution

Reply:
@spirosoik We should apply a fix in the lenses-cli if we need it

Reply:
@spirosoik lol yes i know....

as an aside why is it we are not just injecting the .properties files directly in the helm chart?